sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/DeviceInfoHelper.java

```diff
@@ -17,7 +17,6 @@
 import android.util.DisplayMetrics;
 import android.view.Display;
 import android.view.Surface;
-import android.view.WindowManager;
```

Comment:
Do we need this change?

Reply:
I think yes, because this import wasn't used.